utils/hash_tools.go

```diff
@@ -54,12 +54,13 @@ func CalcFileCRC32(filePath string) uint32 {
 
 // CalcFileHash
 // @notice keccak256(md5(file))
-func CalcFileHash(filePath string) string {
+func CalcFileHash(filePath, encryptionTag string) string {
 	if filePath == "" {
 		Log(errors.New("CalcFileHash: missing file path"))
 		return ""
 	}
-	return hex.EncodeToString(crypto.Keccak256(CalcFileMD5(filePath)))
+	data := append([]byte(encryptionTag), CalcFileMD5(filePath)...)
```

Comment:
this might be conflict with @qinyuanwen 's CID pr, need to coordinate

Reply:
It seems easy enough to integrate. I just need to add the encryption tag to the result of getFileData from his PR. Shouldn`t be a problem